build.assets/Dockerfile

```diff
@@ -282,5 +327,22 @@ RUN cd /usr/local/lib && \
     ldconfig
 COPY pkgconfig/buildbox/ /
 
+# Install JS gRPC tools
+RUN (npm install --global grpc_tools_node_protoc_ts@5.0.1)
+
+FROM buildbox AS grpc_node_plugin_binary_prebuilt
+ONBUILD RUN (npm install --global grpc-tools@1.11.2)
+
+FROM buildbox AS grpc_node_plugin_binary_compiled
+COPY --from=grpc_node_plugin grpc_node_plugin ./grpc_node_plugin
+ONBUILD RUN npm install --global --ignore-scripts grpc-tools@1.11.2 && \
+    mv grpc_node_plugin $(npm root -g)/grpc-tools/bin/. && \
+    # grpc-tools needs protoc but we already install it a couple of steps above.
+    ln -s $(which protoc) $(npm root -g)/grpc-tools/bin/protoc
+
+# A "conditional" FROM like this breaks up the number of steps in the progress bar of the given
+# image, so let's use it at the end. This way the progress bar for buildbox stays mostly untouched.
+FROM grpc_node_plugin_binary_${GRPC_NODE_PLUGIN_BINARY_TYPE} as buildbox
```

Comment:
This conditional FROM is based on this SO answer: https://stackoverflow.com/a/60820156/742872